config/generate_export_header.bzl

```diff
@@ -121,6 +143,8 @@ def generate_export_header(
         name = "__%s_export_h" % lib
     if out == None:
         out = "%s_export.h" % lib
+    if export_import_condition == None:
+        export_import_condition = "%s_EXPORTS" % lib.upper()
```

Comment:
Just a minor remark: CMake does not uppercase the `<lib>_EXPORTS` define.

Reply:
Fixed. Thanks again!